lib/Doctrine/DBAL/Driver/OCI8/OCI8Statement.php

```diff
@@ -300,6 +301,7 @@ public function bindParam($column, &$variable, $type = ParameterType::STRING, $l
         $column = $this->_paramMap[$column] ?? $column;
 
         if ($type === ParameterType::LARGE_OBJECT) {
+            /** @var OCI_Lob $lob */
```

Comment:
PHPStan stubs false-positive?

Reply:
Probably.

Reply:
It's actually `OCI-Lob`.

Reply:
hmm, that's not a valid class name 🙈 moving to ignore rather than typehinting _hack_